src/opengl/Sphere.hh

```diff
@@ -7,8 +7,7 @@ private:
   std::vector<uint32_t> ind;
   void addRect(uint32_t i1, uint32_t i2, uint32_t i3, uint32_t i4);
   void addTri(uint32_t i1, uint32_t i2, uint32_t i3);
-  void addVert(float x, float y, float z); // add just coords
-  void addVert(float x, float y, float z, float u, float v); // add with texture
```

Comment:
why get rid of the textured addVert?

This does not appear to break Globe or SolarSystem, which are the only 3D demos I can think of. So we may be fine to keep this.